core/pkg/model/flag.go

```diff
@@ -3,20 +3,13 @@ package model
 import "encoding/json"
 
 type Flag struct {
-	State          string          `json:"state"`
-	DefaultVariant string          `json:"defaultVariant"`
-	Variants       map[string]any  `json:"variants"`
-	Targeting      json.RawMessage `json:"targeting,omitempty"`
-	Source         string          `json:"source"`
-	Selector       string          `json:"selector"`
-	Metadata       Metadata        `json:"metadata"`
-}
-
-type Metadata struct {
-	ID             string `json:"id"`
-	Version        string `json:"version"`
-	FlagSetID      string `json:"flagSetId"`
-	FlagSetVersion string `json:"flagSetVersion"`
+	State          string                 `json:"state"`
+	DefaultVariant string                 `json:"defaultVariant"`
+	Variants       map[string]any         `json:"variants"`
+	Targeting      json.RawMessage        `json:"targeting,omitempty"`
+	Source         string                 `json:"source"`
+	Selector       string                 `json:"selector"`
+	MetaData       map[string]interface{} `json:"metadata"`
```

Comment:
```suggestion
	Metadata       map[string]interface{} `json:"metadata"`
```

Update all references too please 🙏

Reply:
Done